components/desktop/KnitArticleList.vue

```diff
@@ -1,17 +1,79 @@
 <template>
-  <div class="article-list">
-    <knit-article-card/>
-    <knit-article-card/>
-    <knit-article-card/>
-    <knit-article-card/>
-    <knit-article-card/>
-  </div>
+  <section class="article-list article-list--big-main-post">
+    <h2 class="visualy-hidden">Artykuły KNIT</h2>
+    <knit-article-card v-for="(article, index) in articles" :key="index"
+                       :title="article.title"
+                       :author="article.author"
+                       :content="article.content"
+                       :article-thumbnail-url="article.articleThumbnailUrl"
+                       :author-avatar-url="article.authorAvatarUrl"
+                       :published-at="article.publishedAt"
+                       :number-of-likes="article.numberOfLikes"
+                       :number-of-comments="article.numberOfComments"
+                       />
+  </section>
 </template>
 
 <script>
 import KnitArticleCard from '~/components/desktop/KnitArticleCard.vue'
 
 export default {
+  data () {
+    return {
+      articles: [
+        {
+          title: 'Rozpoczęła się rejestracja na Mocarne Warsztaty!',
+          author: 'Grzegorz Nowakowski',
+          content: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Laboriosam perspiciatis voluptatibus beatae quam, doloribus minus!',
+          articleThumbnailUrl: '/temporary/card-photo-0.jpg',
```

Comment:
I think that writing prefix like article in article object is really redundant

Reply:
It is unnecessary for now